docs/todo-due-dates.md

```diff
@@ -0,0 +1,178 @@
+- Feature Name: `todo-due-dates`
+- Start Date: `2020-11-09`
+- RFC PR: (leave this empty)
+- Checkup Issue: (leave this empty)
+
+# Summary
+
+[summary]: #summary
+
+This RFC proposes enhancing the todo functionality with due days, allowing authors to add accountability and commitment to fix the todo items by a specified amount of time.
+
+# Motivation
+
+[motivation]: #motivation
+
+The lint todo functionality is intended to seamlessly integrate with existing linting tools, allowing the incremental fixing of linting errors on very large projects. However, one undesirable side effect of this is that developers may forget about fixing the todo items, leading to the accumulation of technical debt.
+
+By adding a day threshold to the todo items, developers will now have a commitment to fix the linting problems because todo items that are past this threshold will have their severity increased accordingly.
+
+# How We Will Teach This
+
+[pedagogy]: #pedagogy
+
+- There will be documentation on how to configure when todo items are due to be fixed.
+- There will be a config and/or CLI argument to specify the number of days an item's severity changes, either to `warn` or to `error`.
+
+# Requirements
+
+[requirements]: #requirements
+
+- By default, todos are added with no due dates.
```

Comment:
Please define "by default" (aka be more specific). I _think_ you mean when no config is provided.